python/lsst/ts/aos/utils/m1m3/m1m3_aav.py

```diff
@@ -182,9 +188,9 @@ def was_raised(self, start: pd.Timestamp, end: pd.Timestamp) -> bool:
         last_state = self.detailed_states[
             self.detailed_states.index < pd.to_datetime(start, utc=True)
         ]["detailedState"].iloc[-1]
-        if DetailedState(last_state) not in (
-            DetailedState.ACTIVE,
-            DetailedState.ACTIVEENGINEERING,
+        if DetailedStates(last_state) not in (
+            DetailedStates.ACTIVE,
+            DetailedStates.ACTIVEENGINEERING,
         ):
             return False
```

Comment:
Shouldn´t this case be covered by the code below? As in, the last state is also covered by states and if it's not raised the return below should also be false

Reply:
No. This is to test mirror was in the active state before the specified period started. Added a comment about that.